1.table.sql

```diff
@@ -0,0 +1,42 @@
+use CostumeDB
+go
+drop table if exists Sales
+go
+
+create table dbo.Sales(
+    CustomerId int not null identity primary key,
```

Comment:
AS: General rule regarding constraints: It is always smart to use the following syntax for constraints:
ck_TableName_ColumnName_rest of constraint. This way when working with multiple tables and you get a constraint error, it is very simple to understand where the error is coming from.

Reply:
Thanks for that tip, I amended the table.